index.js

```diff
@@ -15,9 +15,11 @@ module.exports = function (str, opts) {
 	var stats = [];
 	var separator = opts.separator || ' ';
 	var reSeparator = new RegExp(escapeStringRegexp(separator), 'g');
+	var headerLength = (lines[0] || '').length;
 
```

Comment:
I'm surprised that made a difference as looping through just 6K items is almost nothing.

> (As a side note, I am using parse-columns for a few of these 6K line outputs per second so it would be too slow for my use cases.)

Interesting. Would love to hear more about what you're using it for :)

-

But yeah. This is fine. Will merge tomorrow. Too tired now.